### DISCUSSION TITLE
Improving bake time for `-stable` releases

### DISCUSSION THREAD

**Discussion by ghost** [BerriAI/litellm#4404] [General]
## Problem 

Releases can have multiple fixes/improvements. Some of which might have unintended regressions - e.g. the recent jwt fix for the ui, lead to login issues for configs without 'general_settings'. 

## Open Question 

How do we release a `-stable` version where fixes have spent enough time (>3d+) baking.

**Comment by ghost** [comment]
cc: @lolsborn @Manouchehri

**Comment by lolsborn** [comment]
I think in order to have a tested -stable release you're going to need something like a -beta tag or -rc that some key folks test out and QA then have a timeline for promoting those.
